Packs/CommonScripts/Scripts/ExtractFQDNFromUrlAndEmail/ExtractFQDNFromUrlAndEmail.py

```diff
@@ -52,16 +52,19 @@ def unescape_url(escaped_url):
     return url
 
 
-def get_fqdn(the_input):
+def get_fqdn(input_url: str) -> str | None:
     fqdn = None
-    domain = get_tld(the_input, fail_silently=True, as_object=True)
+    domain_info = get_tld(input_url, fail_silently=True, as_object=True)
 
-    # handle fqdn if needed
-    if domain:
-        # get the subdomain using tld.subdomain
-        subdomain = domain.subdomain
-        if (subdomain):
-            fqdn = f"{subdomain}.{domain.fld}"
+    if not isinstance(domain_info, Result):
+        raise ValueError(f"Expected to get a Result object but got {type(domain_info)}")
+
+    if domain_info:
+        subdomain = domain_info.subdomain
+        if subdomain:
+            fqdn = f"{subdomain}.{domain_info.fld}"
+        else:
+            fqdn = domain_info.fld
```

Comment:
is this new? I don't see it in the previous version

Reply:
revert